Mathlib/CategoryTheory/Adjunction/Limits.lean

```diff
@@ -67,7 +67,7 @@ Auxiliary definition for `functorialityIsLeftAdjoint`.
 -/
 @[simps]
 def functorialityUnit :
-  𝟭 (Cocone K) ⟶ Cocones.functoriality _ F ⋙ functorialityRightAdjoint adj K where
+    𝟭 (Cocone K) ⟶ Cocones.functoriality _ F ⋙ functorialityRightAdjoint adj K where
     app c := { hom := adj.unit.app c.pt }
```

Comment:
```suggestion
  app c := { hom := adj.unit.app c.pt }
```
(many times in this file, if you're up for it)

Reply:
will go through that file...

Reply:
done